config_test.go

```diff
@@ -4,9 +4,33 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/send"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 )
 
+const (
+	testDir      = "config_test"
+	testSettings = "evg_settings.yml"
+)
+
+// TestConfig creates test settings from a test config.
+func testConfig() *Settings {
+	file := filepath.Join(FindEvergreenHome(), testDir, testSettings)
+	settings, err := NewSettings(file)
+	if err != nil {
+		panic(err)
+	}
+
+	if err = settings.Validate(); err != nil {
+		panic(err)
+	}
+
+	return settings
+}
```

Comment:
Is it reasonable to merge this with the testutils.TestConfig() method? (maybe my moving it there.)

Reply:
Once this moves to a different package, the function will go away (it's just here for now to avoid an import cycle)